frame/support/procedural/src/benchmark.rs

```diff
@@ -773,7 +906,10 @@ fn expand_benchmark(
 							#verify_stmts
 						)*
 					}
-					Ok(())
+					match #impl_last_stmt {
+						Ok(_) => Ok(()), // discard custom Result<(), BenchmarkError> type in this context
```

Comment:
Now that we've changed the `Ok` value, it's always going to be a `()`, so we don't really need to match here.

Reply:
fixed, good cach